### PR TITLE
New test/team/test_team_distribute_default_none.c

### PR DESCRIPTION
**Pull request by spophale**

### CODE REVIEW

File: tests/5.0/teams/test_teams_distribute_default_none.c [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  OMPVV_WARNING("Test only uses default(none) clause and does not guarantee that the default(none) is enforced.");
+
+  for (int x = 0; x < N; ++x) {
+    a[x] = 1;
+    b[x] = x;
+    c[x] = 2*x;
+    d[x] = 0;
+  }
+
+#pragma omp teams distribute num_teams(4) default(none) shared(a, b, c, d, num_teams) private(privatized)
```
Comment (jhdavis8) [inline tests/5.0/teams/test_teams_distribute_default_none.c:38]: Wondering if this is compliant...
4.5: "If specified, a teams construct must be contained within a target construct. That target construct must not contain any statements or directives outside of the teams construct."

5.0: "A teams region can only be strictly nested within the implicit parallel region or a target region. If a teams construct is nested within a target construct, that target construct must contain no statements, declarations or directives outside of the teams construct."

Comment (spophale) [reply tests/5.0/teams/test_teams_distribute_default_none.c:38]: I don't understand the comment. We are not using the target construct. IF it was nested then the 5.0 part would apply. This is a standalone teams construct.

Comment (spophale) [reply tests/5.0/teams/test_teams_distribute_default_none.c:38]: @jhdavis8 @tmh97 review complete ?

Comment (jhdavis8) [reply tests/5.0/teams/test_teams_distribute_default_none.c:38]: Ah I see, I think I was mistinterpreting the spec to say standalone teams was not permitted. Will approve.

**Comment by spophale** [comment]
Ping @tmh97

**Pull request merged**